config/locales/views/announcements/hi.yml

```diff
@@ -0,0 +1,7 @@
+hi:
+  announcements:
+    learn_more: "और अधिक जानें"
+    edit_announcement_heading: "घोषणा संपादित करें"
```

Comment:
As we're using एडिट term in the general/hi.yml, it would be better if we keep it uniform all across the website. So instead of संपादित we should use एडिट.